README.md

```diff
@@ -55,7 +55,9 @@ This will create:
 * `enabled` (default: _yes_) - whether the service should be enabled
 * `masked` (default: _no_) - whether the service should be masked
 * `state` (default: _started_) - state the service should be in - set to
-  `absent` to remove the service.
+* `absent` to remove the service.
```

Comment:
Editing mistake; should go to the line above

Reply:
README format should be addressed with e8295ca